novatel_gps_driver/src/nodelets/novatel_gps_nodelet.cpp

```diff
@@ -168,7 +171,12 @@ namespace novatel_gps_driver
   class NovatelGpsNodelet : public nodelet::Nodelet
   {
   public:
-    NovatelGpsNodelet() :
+      uint8_t status_gps;
+      ros::Time last_update_time_;
+      cav_msgs::DriverStatus status_;
+      ros::Duration time_difference;
+      ros::Timer timer;
```

Comment:
All these new variables should be private as external classes do not need to access them.